config/config.go

```diff
@@ -631,7 +631,7 @@ var defaultConf = Config{
 	Experimental: Experimental{
 		AllowAutoRandom: false,
 	},
-	EnableDynamicConfig: false,
```

Comment:
The default value should be changed to `false`

Reply:
fixed. PTAL @rleungx